src/mvba/vcbc/error.rs

```diff
@@ -0,0 +1,23 @@
+use crate::mvba::proposal::Proposal;
+use blsttc::PublicKeyShare;
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq)]
+pub enum Error {
+    #[error("invalid proposer. expected {0:?}, get {1:?}")]
+    InvalidProposer(PublicKeyShare, PublicKeyShare),
+    #[error("invalid proposal: {0:?}")]
+    InvalidProposal(Proposal),
+    #[error("duplicated proposal: {0:?}")]
+    DuplicatedProposal(Proposal),
+    #[error("encoding/decoding error: {0}")]
+    Encoding(String),
```

Comment:
`thiserror` provides macros to automatically generate `From` impls for other errors:
```suggestion
    Encoding(#[from] bincode::Error),
```